assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -484,10 +484,14 @@ che.singleport.wildcard_domain.ipless=false
 ### Experimental properties
 # Next properties are subject to changes and removal, so do not rely on them in a stable Che assembly
 
-# Workspace.Next feature API endpoint. Should be a valid HTTP URL that includes protocol, port etc.
+# Workspace tooling plugins registry endpoint. Should be a valid HTTP URL that includes protocol, port etc.
 # In case Workspace.Next is not needed value 'NULL' should be used
 che.workspace.feature.api=NULL
 
+# Docker image of Che plugin broker app that resolves workspace tooling configuration and copies
+# plugins dependencies to a workspace
+che.workspace.plugin_broker.image=garagatyi/broker:websocket
```

Comment:
eclipse/che-plugin-broker?

Reply:
PR description contains an explanation why I decided to use this value

Reply:
The description explains what but not why. I'm ok to have garagatyi/broker:websocket in WP PR but it makes sense to use eclipse/che-plugin-broker image when we want to merge it to master. 
Or you have an issue with that?

Reply:
Right not used image is working whereas upstream one doesn't work. I'll update it when code in upstream is merged.

Reply:
but you are going to merge with pr with eclipse one? Right?

Reply:
yes, I'll update the PR soon